src/components/Onboarding/Onboarding.vue

```diff
@@ -125,6 +125,8 @@ export default {
       }
     },
     loadTagView(tag) {
+      this.$store.dispatch('setFullTagPageOrigin', '1900s_slide')
+      console.log("dispatch")
```

Comment:
???? why 1900s_slide? What's that and why only this one?

Remove debug log

Reply:
I put 1900 for all collections by year. The origin is currently only used to know if we want to come back to tagslide or to the explorator fromthe full image. Regarding that, the only interesting options are:
1) from the tagsslide -> return direct
2) from collections of 1950 1960 and so on... -> return to explorator.

Reply:
I changed its name for 'colelction_by_decade'